connectors/v2/groovemusic.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = '#player';
+
+Connector.trackArtImageSelector = '#player .imgWrapper .img';
+
+Connector.artistSelector = '#player .secondaryMetadata a:first';
+
+Connector.trackSelector = '#player .primaryMetadata a:first';
+
+Connector.getAlbum = function() {
+	var album = $('#player .imgWrapper .img').attr('alt');
+	album = album.substr(album.indexOf('-')+1).trim();
+	return album || null;
+};
+
+Connector.currentTimeSelector = '.playerDurationText .playerDurationTextOnGoing';
+
+Connector.playButtonSelector = '#player button.iconPlayerPlay';
+
+Connector.isPlaying = function () {
```

Comment:
since you've specified playButtonSelector, this function is not necessary.  You can safely remove either this function or the playButtonSelector.